### ISSUE TITLE
Build for Ubuntu

### ISSUE THREAD

**Issue by asachs01** [sensu/sensu-ruby-runtime#19]
@jspaleta is there any way we can get an Ubuntu build going here?

**Comment by lrosenman** [comment]
I'd love that too, for 16.04 LTS.

**Comment by jspaleta** [comment]
Yeah we probably can.  If its needed. I would have thought the debian one would work.

Adding an Ubuntu build is just a Dockerfile for the specific ubuntu version following the debian9 Dockerfile as a guide.  

Though I'd like to know if the existing Debian9 runtime works on Ubuntu first.

and then updating the build.sh  to include the new Ubuntu platform.

**Comment by jspaleta** [comment]
Okay I'm going to be spinning up a new release with changes to the Debian runtime which I think will help it run on Ubuntu. Hopefully will have something out tomorrow.

**Comment by jspaleta** [comment]
0.0.9 included glibc changes that should make Debian assets compatible with Ubuntu.

0.0.10 release now includes platform test matrix to test glibc and openssl shared library operation.
.
Debian asset is built and tested against multiple Ubuntu LTS releases from 14.04 to 18.04, so we shouldn't need an Ubuntu specific release. Ubuntu 12.04 does not work due to requiring very old glibc verison.

We could generate an ubuntu 12.04 specific asset if required.

**Comment by matthiassb** [comment]
@jspaleta Is it possible to generate  for ubuntu 12.04?

**Comment by matthiassb** [comment]
@jspaleta I executed your debian9 Dockerfile script as a bash script on a clean 12.04 with GLIBC 2.15

Works smoothly and with SSL support. Got some older 12.04 systems added into Sensu and working without any issues

![image](https://user-images.githubusercontent.com/2223937/76595883-2e8b9c80-64d3-11ea-8b02-67391a5d2d30.png)
